resources/assets/lib/shopify-debug.ts

```diff
@@ -36,7 +36,7 @@ export async function fetchAllVariants(): Promise<any[]> {
   return variants;
 }
 
-export async function fetchAllVariantIds(): Promise<{ }> {
+export async function fetchAllVariantIds(): Promise<unknown[]> {
```

Comment:
The explicit return type isn't needed (it's `{ gid: string, name: string }`,  anyway)

Reply:
it is required...

```
TS2314: Generic type 'Promise<T>' requires 1 type argument(s).
```

Reply:
oh I forgot about just removing the typing entirely :raised_eyebrow: